apps/server/Dockerfile

```
@@ -69,15 +71,22 @@ RUN bun build \
     --outfile ./dist/index.js \
     ./apps/server/src/index.ts
 
-FROM oven/bun:alpine AS runner
+FROM docker.io/oven/bun:alpine AS runner
 WORKDIR /app
 
+RUN apk add --no-cache curl
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)